configuration/items.md

```diff
@@ -206,7 +206,46 @@ Number Livingroom_Temperature "Temperature [%.1f °C]"
 The state of an Item depends on the Item type, the Channel bound to the Item, and internal or external events.
 A analogy can be drawn between the state of an Item and the value of a variable in a computer program.
 
-<!-- TODO add a complete description about the actual state of an Item, e.g. Initialization, UNDEF, Binding etc. -->
+{: #item-state}
+#### Item State
+
+This section provides information about what a user can expect regarding the behavior of the state of an Item
+
+-  Items are created with a state of `NULL`.
+The Item will remain in this state until it has been acted upon
+-  Subsequent to the creation of an Item, operations in openHAB such as a user interacting with the Item using the `Basic UI`, or a Binding updating the state of an Item will change the state of the Item
+-  Once an Item is being used, its state will depend upon the Thing to which it is linked, or upon the specifics of the Binding that created the Item.
```

Comment:
Isn't that obvious? If you want to emphasize, that the possible states of an Item depend on the Binding, you should make that clearer

Reply:
Agree.  Will re-write this sentence to say, "An Item's state may also be set through a Binding which may be reacting to changes in the real world."